### DISCUSSION TITLE
VSCode setup

### DISCUSSION THREAD

**Discussion by joe61vette** [idaholab/moose#22390] [Q&A Getting Started] [answered]
Was dismayed to see atom being sunsetted.  But thanks to Moose team for giving us an alternative.

I made it to the "Recommended settings" step of the instructions.  Where and how do I add this to VSCode?  For example, do I save it in a file with a special name? etc.  In VSCode, I can open the settings but there are thousands of things.  I have no idea where to put the provided code.

Thanks,
  Joe Kelly

**Comment by GiudGiud** [comment]
Hi Joe

Yeah in VSCode you need to press: shift cmd P to pull up the settings bar
Then Preferences: Open User Settings 
and paste it there

Best,
Guillaume

**Reply by GiudGiud** [reply]
I ll add it to the docs

**Reply by hugary1995** [reply]
IIRC there are `Open User Settings (UI)`, `Open User Settings (json)`, `Open Workspace Settings (UI)` and `Open Workspace Settings (json)`.

User settings will be the global settings for all projects, while Workspace settings is specific to each project (it will actually save the settings in json format in the `.vscode` folder in the current workspace). Depending on your actual need, you may have to choose between user settings and workspace settings.

UI will open settings inside vscode with some nice user interface, and json is its reflection in json format. They should be interoperable.

**Comment by milljm** [comment, marked as answer]
I always have to refer to past instructions as well. I believe you're after editing your workbench settings for that portion. Straight from my own notes (this is for Macintosh machines. The short cut keys will be different for Linux users:

```pre
edit the setting.json using keys `cmd+,` (command + comma), search for `workbench color`,
Click `Edit in settings.json`, append the following...
```
... the following being what ever it was you are trying to add/or edit to workbench settings, in json format. It's a button at the top right (for me). See attached picture.

<img width="931" alt="Screen Shot 2022-10-13 at 10 57 54 AM" src="https://user-images.githubusercontent.com/6414286/195659082-7a11624b-afe5-4521-82c1-ec87df1fccfd.png">

**Reply by joe61vette** [reply]
The key was finding the "edit JSON Button".

Thanks for the help everyone.
  Joe
